### DISCUSSION TITLE
TiddlyMemo 发行版的翻译

### DISCUSSION THREAD

**Discussion by oflg** [BramChen/tw5-docs#5] [General]
@BramChen 您好！

我是 TiddlyMemo 的作者。感谢您在 https://bramchen.github.io/tw5-docs/zh-Hans/ 上翻译了 [TiddlyMemo by oflg](https://bramchen.github.io/tw5-docs/zh-Hans/#TiddlyMemo%20by%20oflg)。

https://tiddlymemo.org/zh-Hans 里也有翻译好的 ，因为我没有找到在哪里 PR 翻译中文文档，可否帮忙放上去？

😊

**Comment by BramChen** [comment]
Hi Guang Li

https://bramchen.github.io/tw5-docs/zh-Hans/ 是 https://tiddlywiki.com/
與其預覽版綜合的簡體中文翻譯版，若您於
https://github.com/Jermolene/TiddlyWiki5/tree/tiddlywiki-com
提交相關文檔，我便會適時更新至  https://bramchen.github.io/tw5-docs/zh-Hans/ 。

Guang Li ***@***.***> 於 2022年4月21日 週四 上午11:28寫道：

> @BramChen <https://github.com/BramChen> 您好！
>
> 我是 TiddlyMemo 的作者。感谢您在 https://bramchen.github.io/tw5-docs/zh-Hans/ 上翻译了 TiddlyMemo
> by oflg
> <https://bramchen.github.io/tw5-docs/zh-Hans/#TiddlyMemo%20by%20oflg>。
>
> https://tiddlymemo.org/zh-Hans 里也有翻译好的 ，因为我没有找到在哪里 PR 翻译中文文档，可否帮忙放上去？
>
> 😊
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BramChen/tw5-docs/discussions/5>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAH2QUEZVI3S2VDAUQ4V5HLVGDDMHANCNFSM5T54KSEA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Bram Chen
